lua/entities/gmod_wire_expression2/core/table.lua

```diff
@@ -1122,7 +1122,6 @@ registerCallback( "postinit", function()
 			rv1.n[n] = rv2
 			rv1.ntypes[n] = id
 			self.GlobalScope.vclk[rv1] = true
-			return rv2
```

Comment:
[`wire_expression_types2[id][2]`](https://github.com/wiremod/wire/blob/f374d5bbcef78667aaf4a985f50a9553e3c98112/lua/entities/gmod_wire_expression2/core/init.lua#L112) from the looks of it. (Whoever wrote this code obviously hates readability.)